### PR TITLE
I have added the array data structure in cpp file

### PR DESCRIPTION
**Pull request by KadhamVj23**
I have added the  Array data structure in cpp language. 
Please do add the changes and accept my PR.

### CODE REVIEW

File: cpp/.vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (anuragdevon) [inline cpp/.vscode/tasks.json:1]: @KadhamVj23 remove .vscode files

File: cpp/.vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "tasks": [
```
Comment (anuragdevon) [inline cpp/.vscode/tasks.json:2]: @KadhamVj23 also remove the executable files.

File: cpp/array.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// K'th smallest/ Largest Element in unsorted array
```
Comment (anuragdevon) [inline cpp/array.cpp:1]: @KadhamVj23 rename file according to the algorithm that is being implemented. `array.cpp` does not convey the work the program is doing.

File: cpp/array.cpp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    // sort the given array
+    sort(arr, arr+n);
+
+    // return kth element int the sorted array
+    return arr[k-1];
+}
+
+// main code
+int main(){
+    int arr[] = {12, 3, 5, 7, 19};
```
Comment (anuragdevon) [inline cpp/array.cpp:22]: @KadhamVj23 make array input user dynamic.

File: cpp/array.cpp [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    int n = sizeof(arr)/sizeof(arr[0]);
+    int k = 2;
```
[the remark this answers is not in the document]
Comment (KadhamVj23) [reply cpp/array.cpp:24]: Ok I will do it

**Comment by KadhamVj23** [comment]
I was unable to do changes so I deleted this . I want to submit another PR , can I do it

**Comment by anuragdevon** [comment]
@KadhamVj23 ok, you can create a new PR. Closing this PR as mentioned by @KadhamVj23 .

**Pull request closed without merging**